docs/organizations/managing-security-and-risk.md

```diff
@@ -1,73 +1,131 @@
+
+For a detailed view of the distribution on a specific week, hover over the graph.
+
+![Security and risk management open finding history panel](images/security-risk-management-overview-history-open.png)<!-- TODO TAROT-2591: Screenshot -->
+
+### Activity history
+
+The **Activity history** panel displays a graph showing weekly counts of open and closed findings over the past three months and the overall open finding trend.
+
+For a detailed view of the counts on a specific week, hover over the graph.
+
+![Security and risk management activity history panel](images/security-risk-management-overview-history-activity.png)<!-- TODO TAROT-2591: Screenshot -->
+
+### Top 10 high-risk repositories
+
+The **Top 10 high-risk repositories** panel displays the repositories with the highest number of open findings, ordered by severity.
+
+!!! note
+    This panel may list less than ten repositories if there are fewer than ten repositories with open findings in the organization or if less than ten repositories are selected in the dropdown filter.
+
+![Security and risk management high risk repositories panel](images/security-risk-management-overview-top-risk.png)<!-- TODO TAROT-2591: Screenshot -->
+
+### Top 10 common security categories
+
+The **Top 10 common security categories** panel displays the most common security categories of open findings, ordered by count.
+
+To access the findings page with the corresponding filter applied, click on a category.
+
+![Security and risk management top categories panel](images/security-risk-management-overview-top-categories.png)<!-- TODO TAROT-2591: Screenshot -->
+
+## Findings {: id="item-list"}
+
+The **Security and risk management findings** page displays a filtered list of findings, sorted by status. Use this page to review and prioritize findings and track the progress of your security efforts.
+
+To access the overview page, select an organization from the top navigation bar, select **Security and risk** on the left navigation sidebar, and click the **Findings** tab in the top bar.
+
+![Security and risk management findings page](images/security-risk-management-findings.png)<!-- TODO TAROT-2591: Screenshot -->
+
+When viewing the findings, you can update the filtering criteria by clicking the **Severity**, **Status**, **Repository**, **Security category**, or **Scan type** drop-downs above the list.
+
+### Viewing a single finding
+
+To view the details of a single finding, click the **Details** column of the finding in the list.
+
+<!-- TODO TAROT-2591: Draft section -->
+
+![Security and risk management single finding page](images/security-risk-management-findings-single.png)<!-- TODO TAROT-2591: Screenshot -->
+
+## Sharing a filtered view of findings {: id="sharing-filtered-view"}
+
+To share the current view of the overview or findings page, click the **Copy URL** button in the top right-hand corner of the page. This action copies the URL with the current filters applied to the clipboard.
+
```

Comment:
Consider adding a screenshot of the screen area scoped to the button.

Reply:
I'll leave the screenshot out for now, since we're telling the user where to look for it. Open to reviewing later.